rfcs/protocols/discovery/readme.md

```diff
@@ -4,5 +4,16 @@
  SPDX-License-Identifier: CC-BY-4.0
  -->
 # Discovery Protocol
+* Authors: Bishakh Chandra Ghosh, Venkatraman Ramakrishna, Krishnasuri Narayanam
+* Status: Draft
+* Since: 25-July-2021
 
-TODO
+
+
+
+When a network ("local network" from here on) tries to configure the idetntity of another network ("foreign network" from here on) for interoperation, the broad steps involved are (i) Network Discovery (ii) Network Identity Validation. Both these steps are carried out by the IIN agent of the participants of the local network.
```

Comment:
idetntity -> identity 

there are a number of other typos, that I haven't pointed out. I suggest running a spell checker across all the docs